src/components/measure/MeasureDirective.js

```diff
@@ -43,7 +43,7 @@ goog.require('ga_measure_service');
                 update(evt.target);
               });
             });
-            newFeature.changed();
+            update(newFeature);
```

Comment:
don't know  why the first call to `changed()` is ignored, the 2nd trigegrs correctly the event.